### ISSUE TITLE
Publishing to NPM

### ISSUE THREAD

**Issue by qfox** [bem/bem-core#1285]
Can we publish releases to NPM? It should be viable since npm3.0 dedupe packages by default

**Comment by awinogradov** [comment]
👍

**Comment by tadatuta** [comment]
yep, and `bem-components` with precompiled styles as well. will do it as soon as find some time.

**Comment by veged** [comment]
Resolved: package is published to NPM as bem-core@5.0.0.
